django/contrib/contenttypes/fields.py

```diff
@@ -359,10 +359,43 @@ def resolve_related_fields(self):
         self.to_fields = [self.model._meta.pk.name]
         return [(self.remote_field.model._meta.get_field(self.object_id_field_name), self.model._meta.pk)]
 
+    def _get_path_info_with_parent(self):
+        """
+        Returns the path with joining through the parent model.
```

Comment:
We are using PEP257 style for new docstrings "Return ..."
I'm having trouble understanding "with joining through the parent model" -- is it correct?